e2e/fixtures/legalCounsellors.js

```diff
@@ -0,0 +1,10 @@
+const config = require('../config.js');
+module.exports = {
+  legalCounsellor: {
+    firstName: 'Ted',
+    lastName: 'Robinson',
+    telephone: '02324000000',
+    organisation: 'London Borough Hillingdon',
```

Comment:
note that on aat this probs won't work

Reply:
I have no idea how the search works there tbh, if you search for just hillingdon it will work fine but when you add the entire thing suddenly it can't be found.

Reply:
Ha! Will look into a way around it. Thanks.

Reply:
We use the same component in the child solicitor test, might be useful as an example

Reply:
Done! Thanks.